### ISSUE TITLE
get position of obj

### ISSUE THREAD

**Issue by santolucito** [Barnard-PL-Labs/MaxPy#2]
is there a way to get the positon of an existing obj? It seems like this might be relevant 

https://github.com/Barnard-PL-Labs/MaxPy/blob/cbc57fef020f87e1fd61ca663b1cc204691b3f5a/maxpy/tools/patchfuncs/placing.py#L367

**Comment by ryurongliu** [comment]
yeah, should be an easy function to write - it's in the object's JSON dict under "patching_rect", which has a list of four values [x, y, width, height], so getting position would be something like:
obj._dict["patching_rect"][0:2]
